forc/src/cli/commands/list.rs

```diff
@@ -0,0 +1,30 @@
+use crate::cli::ListCommand;
+use crate::utils::plugin_descriptions::plugin_description;
+use anyhow::Result;
+use clap::Parser;
+use std::path::PathBuf;
+
+
+#[derive(Debug, Parser)]
+pub struct Command {}
+
+pub(crate) fn exec(command: ListCommand) -> Result<()> {
+    let ListCommand {} = command;
+
+    for path in crate::cli::plugin::find_all() {
+        print_plugin_and_description(path);
+    }
+    Ok(())
+}
+
+fn print_plugin_and_description(path: PathBuf) {
```

Comment:
Obviously very similar to what is used by `plugins` command -- but with a tab separated description